utils/remote.cc

```diff
@@ -2446,6 +2451,10 @@ static int processArgs(char const* rpcurl, int argc, char const* const* argv)
                 debug = true;
                 break;
 
+            case 968: /* Unix domain socket */
+                unix_socket_path = tr_strdup(optarg);
```

Comment:
One thing to consider is total path length, which on linux is 108 and on darwin is 104. I think even clients need a check to make sure they can connect to the socket.

Reply:
What has a limit?

Reply:
Unix socket paths are very limited for historical reasons. You cannot call `bind()` on them when the limit is over their max length. See [`<sys/un.h>` docs](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/sys_un.h.html#tag_13_67_04).

Reply:
https://curl.se/libcurl/c/CURLOPT_UNIX_SOCKET_PATH.html
Can't the this function call handle it?
It fails when i write a large path.

Reply:
As long as it fails with a discernible error then that's fine.